src/main_thread/init/multi_thread_content_initializer.ts

```diff
@@ -182,6 +185,7 @@ export default class MultiThreadContentInitializer extends ContentInitializer {
       },
       { clearSignal: this._initCanceller.signal, emitCurrentValue: true },
     );
+    this.trackDebugElementChanges();
```

Comment:
This wouldn't work if the debug element is shown during the content's playback no? (which seems to be most of the time on Canal+'s main application)

Reply:
It's working if the debug element is shown during content playback because this method track a `SharedReference` that represent if the debug element is in use or not.

Reply:
This code has been removed